db/migrate/20170629152622_add_deleter_to_escort.rb

```diff
@@ -0,0 +1,7 @@
+class AddDeleterToEscort < ActiveRecord::Migration[5.0]
```

Comment:
If you're using one this columns to scope the results of a query, should you consider to have an index on it?